src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Socketer/TCPNetworkConnection.cs

```diff
@@ -96,15 +82,12 @@ public void StopConnectionAttempts()
             socketerClient.StopConnectionAttempts();
         }
 
-        /// <summary>
-        /// Call to send data to this endpoint
-        /// </summary>
-        /// <param name="data">data to send</param>
+        /// <inheritdoc />
         public void Send(byte[] data)
         {
             if (!IsConnected)
             {
-                Debug.LogWarning("Attempted to send message to disconnected SocketEndpoint.");
+                Debug.LogWarning("Attempted to send message to disconnected TCPNetworkConnection.");
```

Comment:
>TCPNetworkConnection [](start = 76, length = 20)

Could use string interpolation with {nameof(TCPNetworkConnection)} here for future refactoring